jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/questionForm.vm

```diff
@@ -40,6 +40,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
       <form action="${request.contextPath}/topics/question/new?branchId=${topicDto.topic.branch.id}"
            method="POST" id="topicDto" class="well anti-multipost submit-form" enctype="multipart/form-data">
     #end
+      <input type="hidden" id="branchId" value="${topicDto.topic.branch.id}"/>
```

Comment:
Mmm.. Why do we need it now? Doesn't seem like something should've changed on the page. Could you add a comment if it's needed?